src/rs.cpp

```diff
@@ -236,12 +236,17 @@ int rs2_device_hub_is_device_connected(const rs2_device_hub* hub, const rs2_devi
 }
 HANDLE_EXCEPTIONS_AND_RETURN(0, hub, device)
 
-rs2_device_list* rs2_query_devices(const rs2_context* context, rs2_error** error) BEGIN_API_CALL
+rs2_device_list* rs2_query_devices(const rs2_context* context, rs2_error** error)
```

Comment:
the BEGIN_API_CALL and HANDLE_EXCEPTIONS are not present in the new version. Is it on purpose?

Reply:
Yes, I think this is right.